src/components/TextInput/TextInput.tsx

```diff
@@ -58,7 +84,15 @@ const TextInput = React.forwardRef<HTMLInputElement, TextInputProps>(function Te
           {label}
         </InputLabel>
         {icon && (
-          <Icon zIndex={1} position="absolute" size="small" right={4} type={icon} {...iconProps} />
+          <Icon
+            zIndex={1}
+            position="absolute"
+            size="small"
+            left={iconAlignment === 'left' ? -20 : null}
```

Comment:
can we use the same scale here?  Currently:

- 4 = 4  *  4 =  16px
- 20 = 20px

We should do: `-5` instead of `-20`

Reply:
Yeap I know, but `pounce` does not scale negative values accordingly. 
```
<Box m={-4}> => margin: -4px;
```